client/my-sites/checklist/wpcom-checklist/checklist-prompt/task.jsx

```diff
@@ -20,6 +23,19 @@ class ChecklistPromptTask extends PureComponent {
 		onClick: PropTypes.func,
 		title: PropTypes.string.isRequired,
 		translate: PropTypes.func.isRequired,
+		closePopover: PropTypes.func.isRequired,
+	};
+
+	dismissPopup = () => {
+		//Remove query string, too
```

Comment:
I don't see this happening on my end. Is this something missed in the current implementation?